CMDLINE.md

```diff
@@ -154,6 +154,8 @@ Commands:
 Options:
       --version  Show version number                                   [boolean]
   -v, --verbose  Verbose mode                                          [boolean]
+  -n, --network  Network to use
```

Comment:
For customers I would like to go with `main`  `dv3`  (because I want to call this a dev network - for developers to build on top of the content fabric) then I think either `test` or `tv4` for the test network and then `local` for the local qfab (127.0.0.1).

It is important to finalize these names before we expose them officially:

In summary this is what I'm thinking:

- `main` - the main Content Fabric network
- `dv3`  - Content Fabric dev network
- `tv4` - Internal Content Fabric test network
- `local` - Local development environment

Reply:
These values are taken from `Config.js`, e.g.:

```js
const networks = {
  main: "https://main.net955305.contentfabric.io",
  demo: "https://demov3.net955210.contentfabric.io",
  demov3: "https://demov3.net955210.contentfabric.io",
  test: "https://test.net955205.contentfabric.io/config",
  dev: "http://localhost:8008/config?qspace=dev&self",
};
```

So I would need to rename all of these to conform.  Easy to do, but don't want tomato this change without checking.